src/rez/utils/colorize.py

```diff
@@ -222,12 +217,6 @@ def _color(str_, fore_color=None, back_color=None, styles=None):
     if not config.get("color_enabled", False):
         return str_
 
-    # lazily init colorama. This is important - we don't want to init at startup,
```

Comment:
I would keep this comment, or at least the part about not calling `init`.

Reply:
I've put back the comment as a warning in the module. Let me know if that's better.

Reply:
Thanks!